simulator/src/testbench.js

```
@@ -536,7 +536,7 @@ function runSingleSequential(testbenchData, scope) {
  */
 function setInputValues(inputs, group, caseIndex, scope) {
     group.inputs.forEach((input) => {
-        inputs[input.label.trim()].state = parseInt(input.values[caseIndex], 2);
+        inputs[input.label].state = parseInt(input.values[caseIndex], 2);
```

ESLint

[error] Assignment to property of function parameter 'inputs'.

(no-param-reassign)